box_it_script/boxIt.js

```diff
@@ -0,0 +1,72 @@
+function topBorder(topWidth) {
+
+    const line  = "\u2501".repeat(topWidth)
+    const output = "\u250F" + line + "\u2513"
+    return output;
+
+    }
+
+function bottomBorder(bottomWidth) {
+
+    const line  = "\u2501".repeat(bottomWidth)
+    const output = "\u2517" + line + "\u251B"
+    return output;
+
+    }
+
+function middleBorder(width) {
+
+    const line  = "\u2501".repeat(width)
+    const output = "\u2523" + line + "\u252B"
+    return output;
+
+    }
+
+function boxIt(text) {
+
+    const side = "\u2503"
+
+    const outputArray = text.slice(2)
```

Comment:
It's always a good idea to put a condition to do an operation if the argument is given. According to the homework description, `node boxit.js // returns '┏┓\n┗┛'`. So you can do something like
```
if (!text) {
  return ┏┓\n┗┛
}
```